tests/test-st2bot-envvars.js

```diff
@@ -82,15 +88,16 @@ describe("environment variable configuration", function () {
     // Load script under test
     var stackstorm = require("../scripts/stackstorm.js");
     stackstorm(robot).then(function (stop) {
-      expect(info_spy).to.have.been.calledThrice;
+      expect(info_spy.args).length.to.be.above(1);
+      // expect(info_spy).to.have.been.calledWith('Using ST2_API_KEY as authentication. Expiry will lead to bot exit.');
```

Comment:
Do we need this line commented or it was just something temporary?
Comment like that makes it look like unfinished.

Reply:
Fixed, thanks.